safe_transaction_service/tokens/tests/test_price_service.py

```diff
@@ -215,3 +216,20 @@ def test_token_eth_value_mocked(self, kyber_get_price_mock: MagicMock):
                 self.assertEqual(
                     price_service.cache_token_eth_value[(random_address_2,)], 0.0
                 )
+
+    @mock.patch.object(
+        PriceService, "get_underlying_tokens", return_value=[], autospec=True
+    )
+    @mock.patch.object(
+        PriceService, "get_token_eth_value", autospec=True, return_value=1.0
+    )
+    def test_eth_price_from_composed_oracles(
+        self, get_token_eth_value_mock: MagicMock, price_service_mock: MagicMock
+    ):
+        price_service = self.price_service
+        token_one = UnderlyingToken("0xdF5e0e81Dff6FAF3A7e52BA697820c5e32D806A8", 0.5)
+        token_two = UnderlyingToken("0xdF5e0e81Dff6FAF3A7e52BA697820c5e32D806A8", 0.5)
```

Comment:
At least mock something realistic, not the same token on the same percentage 😉

Reply:
Ok, I mocked this pool with the current percentages. https://pools.balancer.exchange/#/pool/0xe7ce624c00381b4b7abb03e633fb4acac4537dd6/